lib/streamlit/config.py

```diff
@@ -350,6 +350,14 @@ def _global_development_mode() -> bool:
     type_=bool,
 )
 
+_create_option(
+    "global.appTest",
+    description="Are we in an app test?",
```

Comment:
Not: I would probably add here that it gets automatically set by the app testing framework.